docs/cli.md

```diff
@@ -27,7 +27,7 @@ Or you use the `gcli.sh` script found in the `clients/cli/bin/` directory to run
 The general structure for running commands with the Gravitino CLI is `gcli entity command [options]`.
 
  ```bash
-  usage: gcli [metalake|catalog|schema|table|column|user|group|tag|topic|fileset] [list|details|create|delete|update|set|remove|properties|revoke|grant] [options]
+  usage: gcli [metalake|catalog|schema|model|table|column|user|group|tag|topic|fileset] [list|details|create|delete|update|set|remove|properties|revoke|grant] [options]
```

Comment:
how about `usage: gcli <resource> <action> [options]` ?

Reply:
Because that only gives a vague suggestion to the user, it's more helpful for them to see the possible entities and actions.